### PR TITLE
Update Dropbox definition to include sync_history.db

### PR DESCRIPTION
**Pull request by sydp**
Add database file procesed by plaso parser (ref: https://github.com/log2timeline/plaso/pull/4052)

### CODE REVIEW

**Comment by joachimmetz** [comment]
@sydp FYI the validation test is failing please check the definition:

```
======================================================================
FAIL: testArtifactDefinitionsValidator (validator_test.ArtifactDefinitionsValidatorTest)
Runs the validator over all the YAML artifact definitions files.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\artifacts\tests\validator_test.py", line 24, in testArtifactDefinitionsValidator
    self.assertTrue(
AssertionError: False is not true : in definitions file: data\cloud_services.yaml
```

File: data/cloud_services.yaml [hunk 1]
```diff
@@ -20,12 +20,14 @@ sources:
     paths:
     - '%%users.appdata%%\Dropbox\*.db*'
     - '%%users.localappdata%%\Dropbox\*.db*'
+    - '%%users.homedir%%/.dropbox/instance1/sync_history.db'
```
Comment (joachimmetz) [inline data/cloud_services.yaml:23]: you'll need to add a separate source type for this

Comment (sydp) [reply data/cloud_services.yaml]: My late night Ctrl+C -V failed me.  I've updated with the correct value and local unit tests appear to work.

File: data/cloud_services.yaml [hunk 2]
```diff
@@ -20,12 +20,14 @@ sources:
     paths:
     - '%%users.appdata%%\Dropbox\*.db*'
     - '%%users.localappdata%%\Dropbox\*.db*'
+    - '%%users.localappdata%%\Dropbox\instance1\sync_history.db'
```
Comment (joachimmetz) [inline data/cloud_services.yaml:23]: instance1 hints that there might be more instances, do you know if this can be the case?

Comment (sydp) [reply data/cloud_services.yaml]: An instance2 folder is created when you log out and log in with a different account.   

I've updated the definition to glob after ```instance```.

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**